src/ansys/geometry/core/designer/component.py

```diff
@@ -406,7 +406,11 @@ def set_shared_topology(self, share_type: SharedTopologyType) -> None:
     @check_input_types
     @ensure_design_is_active
     def extrude_sketch(
-        self, name: str, sketch: Sketch, distance: Union[Quantity, Distance, Real]
+        self,
+        name: str,
+        sketch: Sketch,
+        distance: Union[Quantity, Distance, Real],
+        direction: str = "+z",
```

Comment:
I would prefer to have a boolean here instead of a string, but not sure how would I do it while keeping the usage clear

Reply:
Feedback @kmahajan-cadfem ? What's your take?

Reply:
I think its fine with the string, boolean would be confusing and is an abstraction.
on the other hand, passing a string where there are only handful possible choices cries out for `enum` to me, because then user has to provide one or the other and not anything.

in your opinion, if that is overkill for such a small change, then having a possibility is fine at the moment for me 👍🏻

Reply:
Ha... I like the enum option. I think we should accept both string and enum. I can work on that.

Reply:
That is fine with me

Reply:
Implemented in https://github.com/ansys/pyansys-geometry/pull/1042/commits/aeb117be653fbfa6bc88390abe30a0071cf4663b ! Let me know your thoughts